src/pickup.cpp

```diff
@@ -95,7 +98,16 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
         selectmenu.addentry( CONTROL, true, 'v', _( "Control vehicle" ) );
     }
 
-    if( from_vehicle ) {
+    for( auto e : veh->get_parts( "WASHING_MACHINE" ) ) {
+        washing_machine_on = e->enabled;
```

Comment:
Save index of the washmachine here.
You get it with `veh->part_with_feature( veh_root_part, "WASHING_MACHINE" )` above, but then you drop it after comparison.